packages/suite/src/actions/suite/windowActions.ts

```diff
@@ -0,0 +1,12 @@
+import { WINDOW } from './constants';
```

Comment:
I agree, it would be nice to refactor it using redux toolkit, but no time for that